test/test-fixture.html

```diff
@@ -60,22 +60,32 @@
       <x-custom></x-custom>
     </template>
   </test-fixture>
+  <test-fixture id="NestedRepeatFixture">
+    <template>
+      <template id="repeater" is="dom-repeat">
+        <div>{{item}}</div>
+      </template>
```

Comment:
Doesn't a dom-repeat require bound `items` in order to actually repeat something? Where do you set the items?